python/mxnet/numpy/multiarray.py

```diff
@@ -778,13 +778,18 @@ def topk(self, *args, **kwargs):
         """
         raise AttributeError('mxnet.numpy.ndarray object has no attribute topk')
 
-    def argsort(self, *args, **kwargs):
+    def argsort(self, axis=-1, kind='quicksort', order=None):   # pylint: disable=arguments-differ
```

Comment:
I think if `kind` is not supported at all you should default it to something like None.

Reply:
Default 'quicksort' is used to catch the input in case the user input the numpy default for kind